tests/integration/server/users_api_tests.py

```diff
@@ -13,23 +13,27 @@
 # limitations under the License.
 # ------------------------------------------------------------------------------
 """Validating User Account Creation API Endpoint Test"""
+import time
 import requests
+import rethinkdb as r
 from tests.utilities import delete_user_by_username, insert_user
+from rbac.providers.common.db_queries import connect_to_db
 
 
 def test_valid_unique_username():
-    """ Testing the creation of two users with different usernames. """
+    """ Testing the creation of two users
```

Comment:
I think this test only creates one user, can you update the docstring?

Reply:
modified the docstring to - "Testing the creation of an user with create user API."